assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -104,6 +104,7 @@ function Title({
   isSecondary,
   secondaryAction,
   isResizable,
+  displayCollapse = true,
```

Comment:
Could we call it `canCollapse` or `hasCollapse`? `display` sounds like a function here.